hasher.go

```diff
@@ -124,16 +124,24 @@ func (th *trieHasher) digestInnerNode(leftData, rightData []byte) (digest, value
 // digestSumNode returns the encoded leaf node data as well as its hash (i.e. digest)
 func (th *trieHasher) digestSumLeafNode(path, data []byte) (digest, value []byte) {
 	value = encodeLeafNode(path, data)
+	firstSumByteIdx, firstCountByteIdx := GetFirstMetaByteIdx(value)
```

Comment:
```suggestion
	firstSumByteIdx, firstCountByteIdx := getFirstMetaByteIdx(value)
```
I feel this shouldn't be exported